### ISSUE TITLE
between dates

### ISSUE THREAD

**Issue by mstruensee** [react-bootstrap-table/react-bootstrap-table2#59]
filter option for DateBetween to prompt for 2 dates?  >= Date1 and <= Date2

if this does not exist ...
there should be no need to prompt for comparators unless you want >= vs > and <= vs <.  default to both and allow to override to just use 1 of the option?

if it does exist ...
please send me in the right direction :)

thanks

**Comment by AllenFang** [comment]
I'll consider to implement it for data/number filter, thanks.

**Comment by AshwinTayson** [comment]
@AllenFang Is this implemented? Like if I supply a custom date range filter, it comes the cell value between the start and end date?

**Comment by Yusuf023** [comment]
Hi, I am also looking for this implementation. @AshwinTayson, did you find a way to achieve that?

**Comment by AshwinTayson** [comment]
> Hi, I am also looking for this implementation. @AshwinTayson, did you find a way to achieve that?

Yes we went ahead with a custom date range filter using filterRenderer
